css/style.css

```diff
@@ -503,9 +538,9 @@ button.button-inline:hover {
 	margin-bottom: 10px;
 }
 
-#card-dates span {
+#card-meta #duedate {
```

Comment:
it's usually not a good idea to used ID as a CSS selector. I would recommend adding a class and applying style to it.